zendesk_apps_support.gemspec

```diff
@@ -17,6 +17,8 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'image_size'
   s.add_runtime_dependency 'erubis'
   s.add_runtime_dependency 'jshintrb', '~> 0.3.0'
+  s.add_runtime_dependency 'babel-transpiler'
+  s.add_runtime_dependency 'therubyracer'
```

Comment:
it depends on [execjs](https://github.com/babel/ruby-babel-transpiler/blob/master/babel-transpiler.gemspec#L22) if you want to not force people to install therubyracer then just go with that, and install therubyracer in our private consumer of this gem.